pkgs/applications/misc/sdcv/default.nix

```diff
@@ -1,7 +1,9 @@
-{ stdenv, fetchurl, pkgconfig, glib, gettext }:
+{ stdenv, fetchurl, pkgconfig, glib, gettext, fetchFromGitHub, cmake,
```

Comment:
`fetchFromGitHub` should replace `fetchurl` since you are not using the latter anymore